### DISCUSSION TITLE
Can someone suggest a profile for upscaling videos to 60 FPS?

### DISCUSSION THREAD

**Discussion by kastentop2005** [Blinue/Magpie#893] [Q&A] [answered]
I'm noob in this

**Comment by hauuau** [comment, marked as answer]
Do you need to upscale videos on playback or do you want to reencode them?

Magpie is usually more appropriate for upscaling interactive software like games. It's not impossible to use it to upscale a video player window but it's not the most appropriate solution and there might be a bunch of pitfalls if a player rescales video before displaying.

For video playback you should probably checkout mpv with GLSL shaders or VLC with libplacebo integration and custom shaders.
For video upscaling on reencoding there is libplacebo integration in FFmpeg.

**Reply by kastentop2005** [reply]
Ok. Thanks
